subprojects/packagefiles/libjpeg-turbo/meson.build

```diff
@@ -130,11 +127,10 @@ incdir = include_directories('.')
 subdir('simd')
 cdata.set('WITH_SIMD', have_simd)
 if not have_simd
-  simd = static_library('simd', 'jsimd_none.c')
+  simd = []
 endif
```

Comment:
This splits the simd handling needlessly in two different locations. The if block should be deleted and instead in the `simd` subdir you should do this:

```
if simd_opt.disabled()
  simd = []
  subdir_done()
endif
```

Reply:
Updated!